swagger/sdrangel/code/qt5/client/SWGHttpRequest.cpp

```diff
@@ -54,7 +54,7 @@ void SWGHttpRequestInput::add_file(QString variable_name, QString local_filename
 SWGHttpRequestWorker::SWGHttpRequestWorker(QObject *parent)
     : QObject(parent), manager(nullptr)
 {
-    QRandomGenerator::global()->seed(QDateTime::currentDateTime().toTime_t());
```

Comment:
This is generated code so it will be overwritten at next Swagger files generation

Reply:
It is here: https://github.com/f4exb/swagger-codegen/blob/sdrangel/modules/swagger-codegen/src/main/resources/qt5cpp/HttpRequest.cpp.mustache#L48